internal/params/config.go

```diff
@@ -404,6 +404,12 @@ func (c *ChainConfig) IsReceiptLog(epoch *big.Int) bool {
 	return isForked(c.ReceiptLogEpoch, epoch)
 }
 
+// IsAccountAbstraction returns whether num is either equal to the AA fork block or greater.
+func (c *ChainConfig) IsAccountAbstraction(epoch *big.Int) bool {
+	// TODO: Replace with correct block number
+	return isForked(c.IstanbulEpoch, epoch)
```

Comment:
please create a c.AccountAbstractionEpoch param and set it to be TBD.

Reply:
will do